boards/arm/sensor_module/sensor_module.dts

```diff
@@ -12,6 +12,7 @@
 		zephyr,flash = &flash0;
 
 		zephyr,console = &uart4;
+        zephyr,shell-uart = &uart4;
```

Comment:
The indentation is bugging me

Reply:
Think thats just a GitHub bug
<img width="447" alt="image" src="https://github.com/user-attachments/assets/fc9c6a12-006b-499f-a906-1b831e965271" />